lib/block_view_types.go

```diff
@@ -3492,6 +3535,84 @@ func (fe *FollowEntry) IsDeleted() bool {
 	return fe.isDeleted
 }
 
+func (fe *FollowEntry) RawEncodeWithoutMetadata(blockHeight uint64, skipMetadata ...bool) []byte {
+	var data []byte
+	data = append(data, EncodeToBytes(blockHeight, fe.FollowerPKID, skipMetadata...)...)
```

Comment:
Looking through other encoders (e.g. DeSoBalanceEntry, NFTEntry, AssociationEntries, etc), seems as though the standard way we've handled these PKID encodings is through the "EncodeToBytes" helper. We certainly could do a fixed length byte encoding here and save a couple bytes, but it may be better to stick to the helper fn for consistency's sake.